internal/store/fleet.go

```diff
@@ -22,7 +22,9 @@ type Fleet interface {
 	List(ctx context.Context, orgId uuid.UUID, listParams ListParams) (*api.FleetList, error)
 	Get(ctx context.Context, orgId uuid.UUID, name string) (*api.Fleet, error)
 	CreateOrUpdate(ctx context.Context, orgId uuid.UUID, fleet *api.Fleet) (*api.Fleet, bool, error)
+	CreateOrUpdateMultiple(ctx context.Context, orgId uuid.UUID, fleets ...*api.Fleet) error
 	UpdateStatus(ctx context.Context, orgId uuid.UUID, fleet *api.Fleet) (*api.Fleet, error)
+	UpdateStatusMultiple(ctx context.Context, orgId uuid.UUID, fleets ...*api.Fleet) error
```

Comment:
Why does this need to be a method on an interface vs a simple for loop against CreateOrUpdate or UpdateStatus?

Reply:
This wraps all the DB calls in a transaction. its not a simple loop of a call in each iteration.